src/vs/base/common/network.ts

```diff
@@ -129,3 +135,12 @@ class RemoteAuthoritiesImpl {
 }
 
 export const RemoteAuthorities = new RemoteAuthoritiesImpl();
+
+export function toCodeFileUri(path: string, query?: string, skipEncoding?: boolean): string {
```

Comment:
I find it weird that `toCodeFileUri` returns a `string`, shouldn't callers call `toString` as needed and decide on the encoding?